app/lib/experiments.coffee

```diff
@@ -79,101 +72,45 @@ module.exports = class ExperimentServerClient
     @excludedReason = null
     @currentCohort = null
     @currentParticipant = null
-    Geordi.UserGetter.currentUserID = null
-
-  ###
-  when we first get participant, and the user has not started experiment in a previous sessions, we'll need to log it to Geordi
-  ###
-  checkForExperimentStartAndLogIt: (participant) ->
-    if participant.blank_subjects_seen.length==0 && participant.non_blank_subjects_seen.length==0
-      Geordi.logEvent 'experimentStart'
+    @Geordi.UserStringGetter.currentUserID = null
 
   ###
-  when we first discover that a user has ended the experiment, we log it to Geordi
+  Check if current user is eligible to participate in experiments
   ###
-  checkForExperimentEndAndLogIt: (participant) ->
-    if !experimentCompleted && participant? && !participant.active && participant.blank_subjects_available && participant.blank_subjects_available.length==0 && participant.non_blank_subjects_available && participant.non_blank_subjects_available.length==0
-      @experimentCompleted = true
-      Geordi.logEvent 'experimentEnd'
+  shouldGetCohort: (currentUserID) =>
+    @ACTIVE_EXPERIMENT!=null && currentUserID!=@Geordi.UserStringGetter.ANONYMOUS && (!@currentCohort?) && !@experimentCompleted
 
   ###
-  when we get participant, we need to log to Geordi if the user's was excluded
-  ###
-  checkForExcludedAndLogIt: (participant) ->
-    if !@excludedReason? && participant.excluded
-      # if not previously logged, log it.
-      @excludedReason = participant.excluded_reason
-      Geordi.logEvent 'experimentExcluded',participant.excluded_reason
-
+  This method will contact the experiment server to find the cohort for this user in the specified experiment
   ###
-  This method will contact the experiment server to find the participant(experimental data) for this user in the specified experiment
-  ###
-  getParticipant: () ->
```

Comment:
this participant-tracking code was specific to the blanks experiment and is no longer needed.